### ISSUE TITLE
Import Error

### ISSUE THREAD

**Issue by tjkessler** [hgromer/ditto#17]
When importing ditto_lib, I am receiving this error:

```python
>>> import ditto_lib
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\travis\.conda\envs\ditto\lib\site-packages\ditto_lib-1.0.1-py3.7.egg\ditto_lib\__init__.py", line 1, in <module>
    import ditto_lib.itemcollection
  File "C:\Users\travis\.conda\envs\ditto\lib\site-packages\ditto_lib-1.0.1-py3.7.egg\ditto_lib\itemcollection.py", line 29, in <module>
    class ItemCollection:
  File "C:\Users\travis\.conda\envs\ditto\lib\site-packages\ditto_lib-1.0.1-py3.7.egg\ditto_lib\itemcollection.py", line 40, in ItemCollection
    def __eq__(self, other: ItemCollection):
NameError: name 'ItemCollection' is not defined
```

**Comment by tjkessler** [comment]
It looks like functions/methods are created before the class, meaning the ItemCollection object does not exist when used as a type hint:

https://stackoverflow.com/questions/36286894/name-not-defined-in-type-annotation

```python
def __eq__(self, other: 'ItemCollection'):
```
^ this should work

**Comment by tjkessler** [comment]
Solved via #18
